klog.go

```diff
@@ -1096,6 +1096,17 @@ func (l *loggingT) setV(pc uintptr) Level {
 	return 0
 }
 
+// Writer implements io.Writer interface as a pass-through for klog.
+type Writer struct {
+	logFunc func(args ...interface{})
```

Comment:
with logFunc being private, packages outside of klog would not be able to set the field?